com.ibm.wala.util/src/com/ibm/wala/util/config/ExclusionSpecification.java

```diff
@@ -43,49 +41,49 @@
   }
 
   private Kind kind;
-  private Collection<SetOfClasses> inclusions;
-  private Collection<SetOfClasses> exclusions;
+  private SetOfClasses inclusions;
+  private SetOfClasses exclusions;
   
-  public ExclusionSpecification(Kind kind) {
+  public ExclusionSpecification(Kind kind, SetOfClasses inclusions, SetOfClasses exclusions) {
```

Comment:
Do we want to copy in the contents or add a qualified to the member and/or argument, so they don't change?